service/src/main/resources/application.yml

```diff
@@ -99,6 +99,9 @@ leonardo:
 dockstore:
   baseUri: "https://staging.dockstore.org/"
 
+github:
+  personalAccessToken: "<insert personal access token here>"
```

Comment:
Are we saying that we can use this dummy `personalAccessToken` because even with a non-functioning token is attached to the Bearer, we're still only importing public workflows which don't need a real token to check its status?

Reply:
I think you're right, but that wasn't exactly my intent -- I put this here as a way to test while we wait to wire up with ECM. Once we do that, it could potentially stay as a "local only" config, or we could remove it altogether.